project/Dependencies.scala

```diff
@@ -39,7 +38,8 @@ object Dependencies {
   )
    
   lazy val cli = Seq( 
-    "org.fusesource.jansi" % "jansi" % "1.18",
+    "org.jline" % "jline" % "3.19.0",
+    "org.fusesource.jansi" % "jansi" % "2.3.2",
     "org.rogach" %% "scallop" % "4.0.2",
     "net.sf.expectit" % "expectit-core" % "0.9.0" % "it,test"
   )
```

Comment:
Probably. I do like having test stuff at the bottom though, so it's more clear what is a test dep and what is a compile time dep. Maybe we should wait until all dep updates have been merged, since reordering things might cause some conflicts?

Reply:
Note, this one is fine, but reordering others might be best to hold off. Or at least just do those when their deps are updated.